tsconfig.json

```diff
@@ -18,7 +19,8 @@
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
Would you mind explaining why these changes were needed?

Reply:
sure, because we use `import React from 'react'` instead of `import * as React from 'react'`, and that throw that error: `Unable to resolve path to module ‘react’   import/no-unresolved`
BUT this was solved with the setting in `eslintrc`


  ```
settings: {
    'import/resolver': {
      node: {
        extensions: ['.js', '.jsx', '.ts', '.tsx']
      }
    }
  } 
```